src/fosslight_source/_parsing_scanoss_file.py

```diff
@@ -9,6 +9,46 @@
 from ._scan_item import is_exclude_file
 
 logger = logging.getLogger(constant.LOGGER_NAME)
+SCANOSS_INFO_HEADER = ['No', 'Source Name or Path', 'Component Declared', 'SPDX Tag',
+                       'File Header', 'License File', 'Scancode',
+                       'scanoss_matched_lines', 'scanoss_fileURL']
+
+
+def parsing_extraInfo(scanned_result):
+    scanoss_extra_info = []
+    for scan_item in scanned_result:
+        license_w_source = scan_item.scanoss_reference
+        if license_w_source:
+            license_w_source = refine_license_source_output(license_w_source)
+            extra_item = [scan_item.file, license_w_source['component_declared'], license_w_source['file_spdx_tag'],
+                          license_w_source['file_header'], license_w_source['license_file'], license_w_source['scancode'],
+                          scan_item.matched_lines, scan_item.fileURL]
+            scanoss_extra_info.append(extra_item)
+    scanoss_extra_info.insert(0, SCANOSS_INFO_HEADER)
+    return scanoss_extra_info
+
+
+def refine_license_source_input(license_w_source):
```

Comment:
Let's remove this function `refine_license_source_input`.
And set default value in `license_w_source = {}`.
ex. 
`license_w_source = {"component_declared": '', 'file_spdx_tag': '', 'file_header': ''  , ...}`

Reply:
Changed as reviewed.